misc/api-quoter/docs/README.md

```diff
@@ -0,0 +1,159 @@
+# Quoter Tool - Implementation
```

Comment:
The `Quoter Tool ` name doesn't make sense. We need to finalize the name of this tool. IMO, this is a syntax API calls generator. Is this name too long? `Syntax API Calls Generator`